src/coreclr/jit/lsraxarch.cpp

```diff
@@ -1363,7 +1363,7 @@ int LinearScan::BuildBlockStore(GenTreeBlk* blkNode)
             switch (blkNode->gtBlkOpKind)
             {
                 case GenTreeBlk::BlkOpKindUnroll:
-                    if (size < XMM_REGSIZE_BYTES)
+                    if ((size % XMM_REGSIZE_BYTES) != 0)
```

Comment:
Wouldn't the condition `(size % XMM_REGSIZE_BYTES) == 0` only allocate SIMD registers if the block size is a perfect multiple of `XMM_REGSIZE_BYTES`? I don't believe that's the behavior we want since block copies/inits should do as much as possible with SIMD registers before finishing the remainder with GPRs.

Reply:
Yes, never mind. I misread the code.